pkg/agent/sysadvisor/plugin/poweraware/evictor/server/evictor_service.go

```diff
@@ -110,16 +110,9 @@ func (p *powerPressureEvictServer) Start() error {
 }
 
 func (p *powerPressureEvictServer) Stop() error {
-	p.mutex.Lock()
-	defer p.mutex.Unlock()
-
-	if !p.started {
-		general.InfofV(6, "pap: power pressure eviction server already stopped")
-		return nil
-	}
-
-	p.started = false
-	return p.service.Stop()
+	// since there is no resource other than the underlying PluginRegistrationWrapper-guarded grpc service,
+	// which will be cleanup by PluginRegistrationWrapper itself, it has nothing to clean up here
```

Comment:
`PluginRegistrationWrapper` has an internal stop channel that controls the stopping of the server. How do we close the stop channel without calling `Stop` on the wrapper?

Reply:
good point! the current implementation has a recursive reference, which leads to recursive calls of Stop. Will reconstruct code to break the loop.